truss/templates/server/common/truss_server.py

```diff
@@ -88,41 +119,29 @@ def is_binary(request: Request):
         )
 
 
-class TrussServer(kserve.ModelServer):
+class TrussServer:
 
     _endpoints: BasetenEndpoints
     _model: ModelWrapper
     _config: Dict
```

Comment:
We should update these type signatures with all the instance variables now that we don't set them in the parent class.

Reply:
Will do.